applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -17,11 +17,14 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(cis_gateway, CONFIG_LOG_BLE_LEVEL);
 
-#define BLE_ISO_PRES_DELAY_US 10000u
-
 #define BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO                                                  \
-	BT_AUDIO_LC3_PRESET(BT_CODEC_LC3_CONFIG_48_4,                                              \
-			    BT_CODEC_LC3_QOS_10_UNFRAMED(120u, 2u, 20u, BLE_ISO_PRES_DELAY_US))
+	BT_AUDIO_LC3_PRESET(                                                                       \
+		BT_CODEC_LC3_CONFIG(BT_CODEC_CONFIG_LC3_FREQ_48KHZ,                                \
+				    BT_CODEC_CONFIG_LC3_DURATION_10,                               \
+				    LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE),                  \
+				    BT_AUDIO_CONTEXT_TYPE_MEDIA),                                  \
+		BT_CODEC_LC3_QOS_10_UNFRAMED(LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE), 2u,     \
```

Comment:
It is intended that BIS has 4 and CIS has 2 retransmits, right?

Reply:
There's a chance that rtn = 4 will make ACL and ISO link unstable in the before. So rtn =2 is a safe value for dual CIS with 120 bytes payload length.

Reply:
I just kept the values that were already there. BIS was using a preset that set the rtn to 4